src/App.js

```diff
@@ -99,7 +99,8 @@ export default class App extends Component {
         <Header />
         <Filter />
         <Search searchFilter={ this.searchFilter } resetDogs={this.resetDogs} />
-        <CardContainer dogs={ this.state.filteredDogs } adoptableDogs={ this.state.adoptableDogs} allDogs = { this.allDogs }/>
+        <CardContainer dogs={ this.state.filteredDogs } adoptableDogs={ this.state.adoptableDogs} allDogs = { this.allDogs } 
+          rescues={this.state.rescues}/>
```

Comment:
Does your `CardContainer` component truly need all of this state? It seems like a lot to be passing in. Not to mention that some of the data will now be redundant. (e.g. I imagine anything that exists within `filteredDogs` also exists within `allDogs`)